### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by ShadowDaneFoster** [jobinau/pg_gather#51] [General]
@jobinau I just wanted to express my gratitude for your expertise and work and say thank you. I was first introduced to you by a YouTube video you did on PostgreSQL. I have years of PostgreSQL experience and I'm used to how dense concepts can be but everything was explained so clearly in your video. So when you mentioned pg_gather in the video I had to check it out. That was at least a year or more ago. But I've been using it recently and it puts a lot of great information all in one easy to navigate place. I don't always agree with it's recommendations but that's to be expected because it can't know the nuances of a user's workload. So again, thank you.

**Comment by jobinau** [comment]
Hi @ShadowDaneFoster , your words and feedback are so energising. That gives me the power to drive this effort.
Thanks a lot.
